### DISCUSSION TITLE
How to restrict removing a file, if there is a relation somewhere with this file?

### DISCUSSION THREAD

**Discussion by yaroslavyal** [strapi/strapi#7759] [Legacy Topics]
I need to throw an error when the admin user tries to remove the file related to some collection from the admin panel. Please help me. Thanks

**Comment by derrickmehaffy** [comment]
Can you please clarify, this is for the Admin panel correct?

**Comment by lauriejim** [comment]
Hello! can you please give us the exact use-case, this will help us to answer your question.
Thank you.
